scripts/zones/Inner_Horutoto_Ruins/npcs/_5c0_.lua

```diff
@@ -5,36 +5,37 @@
 -- Working 100%
 --  Unable to find EventID for Making Headlines quest. Used dialog ID instead.
 -----------------------------------
-require("scripts/globals/quests");
+require("scripts/globals/quests")
 -----------------------------------
 
-function onTrade(player,npc,trade)
-end;
+function onTrade(player, npc, trade)
+end
 
-function onTrigger(player,npc)
+function onTrigger(player, npc)
     --[[
-    function testflag(set,flag)
+    function testflag(set, flag)
         return (set % (2*flag) >= flag)
     end
 
-    MakingHeadlines = player:getQuestStatus(WINDURST,dsp.quest.id.windurst.MAKING_HEADLINES);
+    local MakingHeadlines = player:getQuestStatus(WINDURST, dsp.quest.id.windurst.MAKING_HEADLINES)
 
-    if (MakingHeadlines == 1) then
-        prog = player:getCharVar("QuestMakingHeadlines_var");
-        if (testflag(tonumber(prog),16) == false and testflag(tonumber(prog),8) == true) then
-            player:messageSpecial(7208,1,dsp.ki.WINDURST_WOODS_SCOOP); -- Confirm Story
-            player:setCharVar("QuestMakingHeadlines_var",prog+16);
+    if MakingHeadlines == 1 then
+        local prog = player:getCharVar("QuestMakingHeadlines_var")
+
+        if testflag(tonumber(prog), 16) == false and testflag(tonumber(prog), 8) then
+            player:messageSpecial(7208, 1, dsp.ki.WINDURST_WOODS_SCOOP) -- Confirm Story
+            player:setCharVar("QuestMakingHeadlines_var", prog + 16)
         else
-            player:startEvent(44); -- "The door is firmly shut"
+            player:startEvent(44) -- "The door is firmly shut"
         end
     else
-        player:startEvent(44); -- "The door is firmly shut"
+        player:startEvent(44) -- "The door is firmly shut"
     end
-    ]]
-end;
+    --]]
```

Comment:
just a heads up, lua ignores that particular change:
```lua
--[[
    stuff
]]
```
```lua
--[[
    same stuff
--]]
```
Both identical in lua's eyes